src/Integrations/Integrations/Laravel/LaravelIntegration.php

```diff
@@ -163,6 +163,29 @@ function ($This, $scope, $args) use ($rootSpan, $integration) {
                     $span->service = $integration->getServiceName();
                     $span->resource = $args[0];
                     $span->meta[Tag::COMPONENT] = LaravelIntegration::NAME;
+
+                    //New user created, assume sign up
+                    if ($span->resource == 'eloquent.created: User') {
```

Comment:
On Laravel 4, this is the only way I found I can detect when a user is created. It does not filter only sign up. Open to listen new approaches

Reply:
From my experience, in 'Vanilla Laravel,' the problem is that the registration process can be completely customized (e.g., with a registration controller responsible for creating a `User`). However, one thing is certain: when a user signs up, their account information is typically stored in a database table or another data source.

I can't think of a way to differentiate between a `User` created as a result of registration and one created through another process...

💭 However, isn't the only one capable of creating a new `User` outside of the registration process the admin? Is that truly an issue?

Reply:
I don't think that's truly an issue neither. This project is aiming to automate user events as much as possible. Developers doing custom approaches will have to still use the SDK.

I was checking with you in case there was a more elegant approach on Laravel 4 but if not, this is the way :)

Thanks for your answer